language/vm/src/file_format_common.rs

```diff
@@ -173,6 +173,8 @@ pub enum Opcodes {
     GET_TXN_SEQUENCE_NUMBER = 0x33,
     GET_TXN_PUBLIC_KEY      = 0x34,
     FREEZE_REF              = 0x35,
+    SHL                     = 0x36,
```

Comment:
Not to be obnoxious, but shouldn't we colocate these with `XOR`/`BIT_OR`/`BIT_AND`?

Reply:
Good question. The answer is yes, but let's not do it right now. If we offset the existing bytecodes this is going to be a breaking change. Sometime before launching we will need to revisit this and formally define the binary encodings for all opcodes altogether.

Reply:
Let me leave a TODO there so we don't forget.